packages/core/useShare/index.md

```diff
@@ -2,13 +2,17 @@
 
 > Reactive [Web Share API](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/share). The Browser provides features that can share content in text or file.
 
+> The `share` method has to be called following a user gesture like a button click. It can’t simply be called on page load for example. That’s in place to help prevent abuse.
+
 ## Usage
 
 ```js
 import { useShare } from '@vueuse/core'
 
-function onShare() {
-  useShare({ title: 'Hello', text: 'Hello my friend!', url: location.href })
+const { share, isSupported } = useShare({ title: 'Hello', text: 'Hello my friend!', url: location.href })
+
+function startShare() {
+  if (isSupported) share()
```

Comment:
The usage needs to be updated. And I think in the example, we don't need to guide users to check `isSupported` again, `isSupported === false` is essentially make `share` a `noop`. `isSupported` could be used in UI guidances or something.